holder.yml

```diff
@@ -10,10 +10,106 @@ info:
     name: GitHub Source Code
     url: https://github.com/w3c-ccg/vc-api
 paths:
+  /credentials/{id}:
+    get:
+      tags:
+       - Credentials
+      summary: Gets a credential or verifiable credential by ID
+      operationId: getCredential
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: Credential retrieved
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: "./components/Credential.yml#/components/schemas/Credential"
+                  - $ref: "./components/VerifiableCredential.yml#/components/schemas/VerifiableCredential"
+        "400":
+          description: Bad Request
+        "401":
+          description: Not Authorized
+        "410":
+          description: Gone! There is no data here
+        "418":
+          description: I'm a teapot - MUST not be returned outside of pre-arranged scenarios between both parties
+        "500":
+          description: Internal Error
+        "501":
+          description: Not Implemented
+    delete:
+      tags:
+       - Credentials
+      summary: Deletes a credential or verifiable credential by ID
+      operationId: deleteCredential
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "202":
+          description: Credential deleted - this is a 202 by default as soft deletes and processing time are assumed
+        "400":
+          description: Bad Request
+        "401":
+          description: Not Authorized
+        "410":
+          description: Gone! There is no data here
+        "500":
+          description: Internal Error
+        "501":
+          description: Not Implemented
+  /credentials:
+    get:
+      tags:
+       - Credentials
+      summary: Gets list of credentials or verifiable credentials
+      operationId: getCredentials
+      parameters:
+        - in: query
+          name: type
+          schema:
+            type: array
+            items:
+              type: string
+              oneOf:
+                - "credentials"
+                - "verifiablecredentials"
```

Comment:
```suggestion
                - "verifiable-credentials"
```

Reply:
Why kebab-case instead of camelCase? We're using camelCase everywhere else, no?

Reply:
its a query string param,  and its pluralized... so i figured it would be better to not have it look like `verifiableCredential` which shows up in VPs, but I am not attached to anything other than fixing the lowercase concatenation issue.